console-api/src/common.rs

```diff
@@ -188,3 +189,31 @@ impl From<&dyn std::fmt::Debug> for field::Value {
         field::Value::DebugVal(format!("{:?}", val))
     }
 }
+
+#[allow(clippy::derive_hash_xor_eq)]
```

Comment:
Ah, this is required because `prost` generates a struct with a `PartialEq` impl, but we can't add `derive(Hash)` to that struct. It might be worth adding a comment explaining why we allow this lint:
```suggestion
// Clippy warns when a type derives `PartialEq` but has a manual `Hash` impl,
// or vice versa. However, this is unavoidable here, because `prost` generates
// a struct with `#[derive(PartialEq)]`, but we cannot add`#[derive(Hash)]` to the
// generated code.
#[allow(clippy::derive_hash_xor_eq)]
```